src/ChangelogGenerator/GitHubPersonal.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ChangelogGenerator;
+
+use function base64_encode;
+use function sprintf;
+
+final class GitHubPersonal implements GitHubCredentials
```

Comment:
I am proposing `GithubUsernamePassword` as class name (or something else), as "personal" is not quite true for oranizations 🤔